msal4j-sdk/README.md

```diff
@@ -16,7 +16,7 @@ Quick links:
 The library supports the following Java environments:
 - Java 8 (or higher)
 
-Current version - 1.13.3
+Current version - 1.0.0-beta.1
```

Comment:
I do not know how the versioning work in Java. But if its dependency system follows the [precedence behavior of Semantic Versioning](https://semver.org/#spec-item-11):

> Example: ... < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0 < ... < 1.13.3"

then a customer who already making a dependency of some recent version of MSAL Java (for example, ">1.12"), they will not pick up our "1.0.0-beta.1". The general practice is to ship the latest experimental behavior in "1.14.0-beta.1".

In other words, stable version and beta versions all share the same versioning naming space, despite their different suffix.

Reply:
Yes, since this was our first beta release, we did not know about the nomenclature. I learned about that after publishing the beta release. will keep this in mind next time. Thanks for reviewing!

Reply:
@g2vinay Do you need this change to happen for testing this feature?

Reply:
@siddhijain 
the version 1.0.0-beta.1 indicates to user that the beta features will GA in 1.0.0 release, plus v1.0.0-beta.1 is already released, we cannot release it again.
For next beta release, I feel it is a good opportunity to correct the beta versioning and release it as v1.14.0-beta.1 and when the feature is ready for stable release, it can be released in v1.14.0